libs/ledger-live-common/src/platform/providers/RemoteLiveAppProvider/index.tsx

```diff
@@ -153,7 +156,7 @@ export function RemoteLiveAppProvider({
       }));
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [allowDebugApps, allowExperimentalApps, providerURL, isMounted]);
+  }, [parameters, isMounted]);
```

Comment:
Better to avoid using `parameters` in the deps array as we don't know if the reference is stable or not
```suggestion
  }, [allowDebugApps, allowExperimentalApps, providerURL, lang, isMounted]);
```